titles/plugin-rhdh/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Configuring plugins in {product}</title>
```

Comment:
```suggestion
<title>Creating plugins in {product}</title>
```
since the procedure is about creating new plugins

Reply:
This tag pertains to the title displayed on the product documentation page, but it's not related to the section title. Let's create titles that align with the comprehensive structure and content of the book.